### DISCUSSION TITLE
[Feature Request] En la sección Admnistrador / Grupos (los 3 tipos) Agregar botón mostrar grupos desactivados

### DISCUSSION THREAD

**Discussion by NicolasPLG** [Cotalker/documentation#339] [3. Ideas & Feature Requests] [answered]
Hola 

Creo que sería bueno tener esa opción en los grupos, para no depender de la base de datos en la activación de éstos.
Contexto: Un cliente tiene 9 SM configuradas, de las cuáles 5 están desactivados (debido a que lo retomaran con el sponsor de cada flujo respectivo). Desde la vista administrados podemos desactivar los grupos, pero no reactivarlos y creo que sería buena idea.

![image](https://user-images.githubusercontent.com/61419471/125817199-1a0061d9-f07f-4d99-9561-e0a756cff5af.png)

Gracias =>

**Comment by acedward** [comment, marked as answer]
Gracias @NicolasPLG está en el roadmap!
Queremos poder desactivar todos los objetos de Cotalker.

Si necesitas priorizar esto por favor contacta tu representante Cotalker.

**Reply by NicolasPLG** [reply]
hola @acedward muchas gracias por la respuesta, mientras está en el roadmap, no es necesario que se priorice.

muchas gracias =>
